src/main/steps/applicant1Sequence.test.ts

```diff
@@ -1,42 +1,74 @@
-import { applicant1Sequence } from './applicant1Sequence';
+import { Sections, applicant1Sequence } from './applicant1Sequence';
+import {
+  APPLICANT_1_CONTACT_DETAILS,
+  APPLICANT_1_DOB,
+  APPLICANT_1_FIND_ADDRESS,
+  APPLICANT_1_FULL_NAME,
+  APPLICANT_1_IDENTITY_DOCUMENTS,
+  APPLICANT_1_MANUAL_ADDRESS,
+  APPLICANT_1_NATIONALITY,
+  APPLICANT_1_OCCUPATION,
+  APPLICANT_1_OTHER_NAMES,
+  APPLICANT_1_SELECT_ADDRESS,
+  APPLYING_WITH_URL,
+  DATE_CHILD_MOVED_IN_URL,
+  FEE_LOOKUP_URL,
+  TASK_LIST_URL,
+} from './urls';
 
 describe('applicant1Sequence', () => {
-  test('should contain 8 entries in applicant 1 screen sequence', () => {
-    expect(applicant1Sequence).toHaveLength(9);
+  test('should contain 13 entries in applicant 1 screen sequence', () => {
+    expect(applicant1Sequence).toHaveLength(13);
 
-    expect(applicant1Sequence[0].url).toBe('/applying-with');
-    expect(applicant1Sequence[0].showInSection).toBe('aboutApplicant1');
-    expect(applicant1Sequence[0].getNextStep({})).toBe('/task-list');
+    expect(applicant1Sequence[0].url).toBe(APPLYING_WITH_URL);
```

Comment:
these hard codings were there for a purpose
if someone changes something in the URLs we won't know if we use constants since test cases will pass
For example
if someone mistakenly updates the applying-with to something else then it might break the application 
We do not have separate tests for URL, this is our place to tests the URLs

Reply:
Resolved